src/apps/loan-list/materials/utils/status-badge.tsx

```diff
@@ -3,7 +3,7 @@ import statusThreshold from "../../../../core/configuration/status-thresholds.js
 import { daysBetweenTodayAndDate } from "../../../../core/utils/helpers/general";
 
 interface StatusBadgeProps {
-  dueDate: string;
+  dueDate: string | null | undefined;
```

Comment:
Is dueDate required here?
If not you could write the same like:
```typescript
dueDate?: string | null;

Reply:
yesss